tests/test_group.rs

```diff
@@ -206,8 +201,15 @@ fn basic_group_setup() {
 #[test]
 /// This test simulates various group operations like Add, Update, Remove in a small group
 ///  - Alice creates a group
-///  - Alice invites Bob
+///  - Alice adds Bob
 ///  - Alice sends a message to Bob
+///  - Bob updates and commits
+///  - Alice updates and commits
+///  - Bob updates and Alice commits
+///  - Bob adds Charlie
+///  - Charlie sends a message to the group
+///  - Charlie updates and commits
+///  - Charlie removes Bob
 fn group_operations() {
     let supported_ciphersuites = vec![
         CiphersuiteName::MLS10_128_DHKEMX25519_AES128GCM_SHA256_Ed25519,
```

Comment:
Please add the P256 ciphersuite. You can also use `Config::supported_ciphersuites` now.